docs-src/_config.yml

```diff
@@ -11,6 +11,7 @@ launch_buttons:
 
 execute:
   allow_errors: false
+  execute_notebooks: auto  # one of ("auto", "force", "cache", "off")
```

Comment:
This is the default, but makes it easier to change it to "off" for testing locally.

Reply:
I agree! `"cache"` is what I would usually use.